TrainingHW4/TrainingHW4/Program.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+
+namespace TrainingHW4
+{
+    class Program
+    {
+        static int FindMinValue(int[] array)
```

Comment:
Simplify `FindMinValue` `FindMaxValue` into one function.
I think you can output min/max values and return array of min/max values.